_includes/head.html

```diff
@@ -31,4 +31,5 @@
   </style>
 <![endif]-->
 
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<meta http-equiv="cleartype" content="on">
+<script async defer src="{{ '/assets/js/main.min.js' | absolute_url }}"></script>
```

Comment:
Re my IMO, that's because HTML5 put the onus of handling spec exceptions on the browsers. If it weren't that way, we'd have conditional comments and object detection for every browser, and a spec full of exceptions. How fun would that be? :)~